src/pages/dashboard.vue

```diff
@@ -425,7 +425,7 @@ const coaEffectiveDate = computed(() => {
             {{ $t('title.section.filingHistory') }}
             <span class="font-normal">({{ filings?.filter(f=>f.displayLedger).length || 0 }})</span>
             <BcrosFilingAddStaffFiling
-              v-if="isStaffAccount"
+              v-if="isAuthorized(AuthorizedActionsE.FILE_AND_PAY)"
```

Comment:
If I understand correctly, this is the "Add Staff Filing" drop-down menu.

I think this should not be controlled by FILE_AND_PAY (a role that regular users will have). I suggest a new role, STAFF_FILINGS, as per row 35 in the [spreadsheet](https://docs.google.com/spreadsheets/d/13P4ZA19lJfXJX95N5OfQMu6jYuotDH6h2hDf2DnAv0E/edit?gid=1874127980#gid=1874127980).

To show this menu, you'll also need to check if these other staff filing are enabled: RESTORATION_FILING and FREEZE_UNFREEZE_FILING.

Reply:
This conversation is outdated, and there's another converation for it. Resolving this one.